src/main/java/Morgan.java

```diff
@@ -0,0 +1,58 @@
+import commands.Command;
+import exceptions.MorganException;
+import javafx.scene.image.Image;
+import parser.CommandParser;
+import storage.Storage;
+import tasks.TaskList;
+
+
+public class Morgan {
+    private static final String EXIT_KEYWORD = "bye";
+    private final TaskList taskList;
+    private final Storage storage;
+
+    private Image userImage = new Image(this.getClass().getResourceAsStream("/images/User.png"));
+    private Image morganImage = new Image(this.getClass().getResourceAsStream("/images/Morgan.png"));
+
+    /**
+     * Constructor for Morgan.
+     *
+     */
+    public Morgan() {
+        this.storage = new Storage();
+        this.taskList = new TaskList();
+    }
+
+    public boolean isExit(String keyword) {
```

Comment:
Perhaps you can consider changing your variable name to a verb like ```hasExited ``` to make it more intuitive to the reader?